### PR TITLE
feat: add describe command for variables command tree

### PR DESCRIPTION
**Pull request by maxwelbm**
- create describe command for variables;
- unit tests;

### CODE REVIEW

File: messages/variables/errors.go [hunk 1]
```diff
@@ -3,8 +3,9 @@ package variables
 import "errors"
 
 var (
-	ErrorGetVariables = errors.New("Failed to describe the origins: %s. Check your settings and try again. If the error persists, contact Azion support.")
-	ErrorFailToDeleteVariable = errors.New("Failed to delete the Variable: %s. Check your settings and try again. If the error persists, contact Azion support")
+	ErrorGetItem                         = errors.New("Failed to describe the variable: %s. Check your settings and try again. If the error persists, contact Azion support.")
+	ErrorMissingArguments                = errors.New("Required flags are missing. You must supply application-id and origin-id as arguments. Run 'azioncli <command> <subcommand> --help' command to display more information and try again")
+	ErrorGetVariables                    = errors.New("Failed to describe the origins: %s. Check your settings and try again. If the error persists, contact Azion support.")
```
Comment (PatrickMenoti) [inline messages/variables/errors.go:8]: failed to describe the variable

File: messages/variables/messages.go [hunk 2]
```diff
@@ -2,25 +2,34 @@ package variables
...
 	// [ variables ]
-  
-	Usage 				= "variables"
-	ShortDescription   = "Manage your variables on the Azion Edge platform"
-	LongDescription    = "Manage your variables' varaibles on the Azion Edge platform"
-	FlagHelp           = "Displays more information about the Rules Engine command"
-	FlagId              = "Unique identifier of the Variable"
+	Usage            = "variables"
+	ShortDescription = "Create variables for edges on Azion's platform"
+	LongDescription  = "Build your Web applications in minutes without the need to manage infrastructure or security"
```
Comment (PatrickMenoti) [inline messages/variables/messages.go:7]: Just copy + paste the same description as the one you used for Short Description. @gabriel-azion will later on edit these messages

**Review by peterpaulo-azion — DISMISSED** [review]

File: messages/variables/messages.go [hunk 3]
```diff
@@ -2,25 +2,34 @@ package variables
...
+	FlagHelp         = "Displays more information about the variables command"
+	FlagId           = "Unique identifier of the Variable"
+
+	// [ describe ]
+	DescribeUsage            = "describe --variable-id <variable_id> [flags]"
+	DescribeShortDescription = "Returns the variable data"
+	DescribeLongDescription  = "Displays information about the variable via a given ID to show the application’s attributes in detail"
```
Comment (PatrickMenoti) [inline messages/variables/messages.go:14]: to show the variable's attributes in detail

File: pkg/cmd/variables/describe/.fixtures/variables.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "uuid": "32e8ffca-4021-49a4-971f-330935566af4",
+    "key": "Content-Type",
+    "value": "json",
+    "secret": false,
+    "last_editor": "maxwelbm@hotmail.com",
```
Comment (PatrickMenoti) [inline pkg/cmd/variables/describe/.fixtures/variables.json:6]: Don't think you should use your actual email here hehe

**Review by pauloSF0 — APPROVED** [review]

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]
change these things please :)

**Review by PatrickMenoti — APPROVED** [review]

**Review by pauloSF0 — APPROVED** [review]

**Pull request merged**